### ISSUE TITLE
Support Homebrew install for MacOS

### ISSUE THREAD

**Issue by eddiewebb** [circleci/local-cli#7]
Although the provided steps to install for non SNAP systems is a brilliantly simple one-liner, I would suggest many, if not most, MacOSX users turn to Homebrew as their default installer.

Supporting homebrew for the local-cli would allow intuitive install of the cli, and is requested by some in CirclelCI community.

https://discuss.circleci.com/t/homebrew-formula/14652/2

The only change needed to this repo is to publish a tarball somewhere that homebrew can download it.  A formula could be created as a `tap` or eventually merged into homebrew-core.

A stable, versioned URL is required for acceptance into homebrew-core - https://github.com/Homebrew/homebrew-core/pull/21661

**Comment by felicianotech** [comment]
The PR for this repo has been merged. Just waiting on Homebrew's end now to close out this Issue.

**Comment by gsaslis** [comment]
FWIW i just installed with `brew install circleci` today.. 

I'm guessing this could be closed?
